documentation/docs/steps/abapEnvironmentCheckoutBranch.md

```diff
@@ -32,8 +32,32 @@ steps:
     abapCredentialsId: 'abapCredentialsId'
     host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
 ```
+
 Please note that the branchName parameter specifies the target branch you want to switch on. Also keep in mind that the repositoryName parameter must define a single repository.
 
+Also you can specify a dedicated file, e.g. `repositories.yml` containing the branches in separate repositories you want to switch on:
+
+```yaml
+steps:
+  abapEnvironmentCheckoutBranch:
+    repositories: 'repositories.yml'
+    abapCredentialsId: 'abapCredentialsId'
+    host: '1234-abcd-5678-efgh-ijk.abap.eu10.hana.ondemand.com'
+```
+
+The associated config file, e.g. `repositories.yml` could look as follows:
+
+```yaml
+repositories:
+- name: '/DMO/GIT_REPOSITORY'
+  branch: 'master'
+- name: '/DMO/GIT_REPO'
+  branch: 'master'
+```
+
+Please note that you need to adapt the file structure according to the structure in this example in order to work.
```

Comment:
I'm not quite sure what you want to say here. Which file structure do I need to adapt?

Reply:
Changed it to `Please note that you need to use the YAML data structure of above example and configure a list of repositories when using the 'repositories.yml' config file.`